doc/memos/rdm0.md

```diff
@@ -0,0 +1,126 @@
+RIOT Maintainers Community (RIOT-MC)  
+Developer Memo: 0  
+Author(s): Emmanuel Baccelli  
+Date: December 2016
+
+***
+RIOT Developer Memo Format, Publishing and Maintenance Process
```

Comment:
Shouldn't this rather be a headline (`# RIOT Developer Memo Format, ...`) and the sections having below secondary headlines?

Reply:
Addressed